pkg/utils/utils.go

```diff
@@ -115,3 +116,22 @@ func CountImportedDataVolumes(dvsDone map[string]bool) int {
 
 	return done
 }
+
+// FormatBytes convert bytes to highest suffix
+func FormatBytes(bytes int64) string {
+	if bytes < 0 {
+		return ""
+	}
+
+	kib := int64(units.KiB)
+	if bytes < kib {
+		return fmt.Sprintf("%d", bytes)
+	}
+	suffix := 0
+	n := int64(kib)
+	for i := bytes / kib; i >= kib; i /= kib {
+		n *= kib
+		suffix++
+	}
+	return fmt.Sprintf("%d%ci", bytes/n, "KMGTPE"[suffix])
```

Comment:
Is it ok to loose precision? It is very visible  with higher numbers i.e. for `1024*1024*1024*1024-1` you get 1023Gi - that is a differnece of 1Gi,

Reply:
Well, right. I think I will round it. I check and in ovirt it's not possible to create VM with memory like `1024*1024*1024*1024-1`. It always rounded to MiB. For storage size it's rounded to KiB.